libsplinter/.env

```diff
@@ -0,0 +1 @@
+DATABASE_URL=postgres://splinter_admin:splinter_test@localhost:5432/splinter
```

Comment:
Because of how we are running the database/migrations I don't think we need this file